alf/bin/train.py

```diff
@@ -76,8 +76,6 @@ def train_eval(root_dir):
 
 def main(_):
     gin_file = common.get_gin_file()
-    if FLAGS.gin_file:
-        common.copy_gin_configs(FLAGS.root_dir, gin_file)
```

Comment:
This change will make continuing training or playing without gin_file fail.

Reply:
should copy gin files to root_dir , `operative.gin`  do not include all operative config actually (sub processes #193 )

Reply:
> should copy gin files to root_dir , `operative.gin` do not include all operative config actually (sub processes #193 )

I've updated the PR description for the motivation. For the subprocess issue, I think we should create a dummy environment inside the main process.

Reply:
> This change will make continuing training or playing without gin_file fail.

Why? If FLAGS.gin_file is None, then nothing happens. `gin_file` will be globbed inside `root_dir`

Reply:
You need to copy inoperative configs